filesystems/blob/src/test/scala/cromwell/filesystems/blob/BlobPathBuilderSpec.scala

```diff
@@ -91,4 +91,17 @@ class BlobPathBuilderSpec extends AnyFlatSpec with Matchers with MockSugar {
     val fileText = (is.readAllBytes.map(_.toChar)).mkString
     fileText should include ("This is my test file!!!! Did it work?")
   }
+
+  ignore should "resolve a path without duplicating container name" in {
```

Comment:
`ignore`?
Also, based on comments in `BlobPathBuilder.scala`, I think I can see what you're checking here, but it could be confusing to a future reader to read this statement about what code shouldn't do instead of what it should do.

Reply:
Yes, ignored because it depends on the local environment being logged in to Azure, so it's useful for dev testing, not for Travis testing. I waffled on whether to include it at all; we still need to work out our approach for testing the "thin ring" of logic that depends on library internals. I ended up leaving it in since there are other tests in this file that follow the same pattern. I never went back and renamed it to something clearer, though, will do that.